<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chefcomment</class>
 <widget class="QDialog" name="Chefcomment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>评价厨师</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/commetimages.jpg</normaloff>:/images/commetimages.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Chefcomment{
background-image: url(:/images/true.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Ume Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>写下评价</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>240</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>请先打分,评语可以为空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>114</red>
         <green>159</green>
         <blue>207</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>114</red>
         <green>159</green>
         <blue>207</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>114</red>
         <green>159</green>
         <blue>207</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;文泉驿等宽微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>做菜效率</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>115</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;文泉驿等宽微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>为TA打分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;文泉驿等宽微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>做菜质量</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
